### ISSUE TITLE
Can this be applied to your own dataset?

### ISSUE THREAD

**Issue by ZouZhengF** [YonghaoXu/Txt2Img-MHN#6]
Can this be applied to your own dataset?

**Comment by YonghaoXu** [comment]
Yes, you can use your own dataset to train this model. Note that the data folder should be structured in a similar way as indicated in the readme file.
